docs/admin/install.rst

```diff
@@ -37,6 +38,9 @@ In order to install SecureDrop Workstation and configure it to use an existing S
   .. note:: A USB stick with a Type-A connector is recommended, as USB-C ports may be disabled on your computer when the BIOS settings detailed below are applied.
 
 - The SecureDrop instance's Admin Workstation and Secure Viewing Station (SVS) USBs, and the full GPG fingerprint of the submission key.
+- The Journalist Workstation USB for the intended user of this workstation, if you intend to use the password manager on SecureDrop Workstation.
```

Comment:
This could probably be an either-or situation, a JW is going to have the necessary credentials for the journalist interface. So something like (but with better wording):

> - A workstation USB:
>   - if the SDW is being set up for a single journalist, use their JW to copy the necessary credentials (JI and optionally passwords)
 >   - if the SDW is being set up for use by multiple jourrnalists, use a JW or admin workstation to copy over the JI details only

Reply:
Yup that makes sense, will take a stab after standup.

Reply:
Upon consideration, we need the Admin Workstation USB during the SecureDrop install verification stage, so it has to be in the inventory no matter what. I'm leaving the existing wording there, but will make the section about copying over the JI details a bit more flexible in terms of which USB is used.